docker/README.md

```
@@ -0,0 +1,377 @@
+# LaChispa Docker Documentation
+
+English | [Español](#español)
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)